dnase/callsnpsMerge.sh

```diff
@@ -217,6 +253,65 @@ for sampleid in `bcftools query -l ${sampleOutdir}/${name}.${mappedgenome}.filte
     #https://genome.ucsc.edu/FAQ/FAQformat.html#format10
 done
 
+echo
```

Comment:
We are going to merge the DELLY and bcftools calls into a single VCF, so shouldn't need to make a separate genotypes file